04/app.js

```diff
@@ -1,8 +1,42 @@
 console.log('DOM');
 
-// struktura do wykorzystania w pętli
-const menuItems = [
-    {text: 'start', url: '/'},
-    {text: 'galeria', url: '/gallery'},
-    {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+// I sposób
+
+const navSection = document.querySelector('nav');
+const newUlSection = document.createElement('ul');
+
+const urlArray = ['/', '/gallery', '/contact'];
+const textArray = ['start','galeria','kontakt'];
+
+if(navSection){
+    navSection.appendChild(newUlSection);
+
+    for(let i=0; i<3; i++){
```

Comment:
Bardziej uniwersalne będzie użycie ilości elementów z tablicy - jak sie ona powiększy to nic więcej nie trzeba zmianić czyli `for(let i=0; i<urlArray.length; i++){`